include/functions/math.hpp

```diff
@@ -6,15 +6,15 @@
 
 #ifndef BIG_INT_MATH_FUNCTIONS_HPP
 #define BIG_INT_MATH_FUNCTIONS_HPP
-
+#include <string>
 
 /*
-    abs
+    big_abs
     ---
     Returns the absolute value of a BigInt.
 */
 
-BigInt abs(const BigInt& num) {
+const BigInt big_abs(const BigInt& num) {
```

Comment:
Why return a `const BigInt` and not a `BigInt`?

Reply:
To be honest, I was just trying things to resolve a problem.